apps/settings/locales/settings.ar.properties

```diff
@@ -476,8 +476,8 @@ wifi-join-password=كلمة السّر
 hotspotSettings-header={{hotspotSettings}}
 ssid-name=اسم الشبكة
 hotspot-open=فتح
-hotspot-wpa-psk=WPA (TKIP)
-hotspot-wpa2-psk=WPA2 (AES)
+hotspot-wpa-psk.innerHTML= &#x202a;WPA (TKIP)&#x202c;
```

Comment:
Better yet, please add the Localization note in settings.en-US.properties.  This is the file that the localizers use as reference.